### PR TITLE
ghmerge and pick-to-branch: further tweak output

### PR DESCRIPTION
**Pull request by DDvO**
Add a log output, a message, and some empty lines to make the overall output more readable.

### CODE REVIEW

**Review by t8m — CHANGES_REQUESTED** [review]

File: review-tools/ghmerge
Comment (DDvO) [reply review-tools/ghmerge:242]: I removed this log output because it was redundant - see lines 266 to268.

Comment (t8m) [reply review-tools/ghmerge:242]: Ah, you're right.

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Needs second review

**Review by mspncp — APPROVED** [review]
LGTM

**Comment by DDvO** [comment]
This appears ready to merge -
can someone with sufficient privilege please do.

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
Merged.  Any idea about the empty `remote:` lines that have appeared?

E.g. from this merge:
>remote: Resolving deltas: 100% (4/4), completed with 4 local objects.
>remote: 
>remote: 
>To github.openssl.org:otc/tools.git

**Comment by t8m** [comment]
> Any idea about the empty `remote:` lines that have appeared?

I suspect they are coming from the GHE hooks.

**Comment by DDvO** [comment]
I cannot find this commit via my upstream repo reference `git@github.com:openssl/tools.git`.
Was it pushed to some other place or which is the current repo address that I should use?

**Comment by t8m** [comment]
Hmm it seems it was somehow lost.

**Comment by t8m** [comment]
Not sure how that happened.

**Comment by t8m** [comment]
OK, pushed again.

**Comment by DDvO** [comment]
Thanks @t8m - now I can see it via my upstream remote `git@github.com:openssl/tools.git`.

**Pull request closed without merging**
